airbyte-ci/connectors/pipelines/README.md

```diff
@@ -408,6 +408,7 @@ This command runs the Python tests for a airbyte-ci poetry package.
 ## Changelog
 | Version | PR                                                         | Description                                                                                               |
 | ------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| 2.6.0   | [#31974](https://github.com/airbytehq/airbyte/pull/31974)  | Merge airbyte-ci-internal into airbyte-ci                                                                 |
```

Comment:
❓ was this necessary to get the binaries working? because of the scripts, or what?

Reply:
Also, we should mention the new command!

Reply:
Yes it was. Basically because the `airbyte-ci` entry point relied on `airbyte-ci-internal` being a separate command. With a binary we can (technically should) only build 1 so we had to merge 2 into 1

Reply:
Got it, makes sense